src/main/java/edu/csula/datascience/acquisition/Source.java

```diff
@@ -14,5 +14,7 @@
  * After storing these objects in memory, then you can download them using
  * Collector#mungee
  */
-public interface Source<T> extends Iterator<Collection<T>> {
+public interface Source<T> extends Iterator<Collection<T>>
+{
+	void getEarthQuakeData();
```

Comment:
First of all, you should not be changing this interface for the sake of adding your own method.